cl/favorites/models.py

```diff
@@ -200,3 +201,30 @@ class Meta:
                 name="favorites_prayer_date_created_user_id_status_880d7280_idx",
             ),
         ]
+
+
+# model to keep track of when we last checked whether a given PACER document was still unavailable for purchase
+class PrayerAvailability(models.Model):
+    recap_document = models.ForeignKey(
+        RECAPDocument,
+        help_text="The document people prayed for.",
+        related_name="prayeravailability",
+        on_delete=models.CASCADE,
+    )
+    last_checked = models.DateTimeField(
+        help_text="The time when the availability of this document was last checked.",
+        default=timezone.now(),
+    )
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["recap_document"],
+                name="unique_recap_document",
+            ),
+        ]
+        indexes = [
+            models.Index(
+                fields=["recap_document", "last_checked"],
+            ),
+        ]
```

Comment:
I don't think we really need that multicolumn index. The queries in this PR primarily use the recap document ID to access the table and we're not using the `last_checked` column for filtering.

However, if the goal with the multicolumn index was to avoid fetching data from both the index and the heap when the table gets big, then I think a covering index would be a better way to go. We could use the UniqueConstraint with the [include](https://docs.djangoproject.com/en/5.1/ref/models/constraints/#include)  argument to do that.  Here's my suggestion: 

```suggestion
        constraints = [
            models.UniqueConstraint(
                fields=["recap_document"],
                name="unique_recap_document",
                include=["last_checked"]
            ),
        ]
```